pkg/conf/conf.go

```diff
@@ -39,8 +39,9 @@ type Main struct {
 	NormalizeRecords  bool
 	LogSpecialRecords bool
 
-	PprofPort uint16
-	PromPort  uint16
+	PprofPort    uint16
+	PromPort     uint16
+	ShortMetrics bool
```

Comment:
Document/add to example config file the new option.

Reply:
Fixed.